peer.go

```diff
@@ -1087,142 +1164,161 @@ func (p *peer) handleInitClosingSigned(req *htlcswitch.ChanClose, msg *lnwire.Cl
 		IsPending:      true,
 	}
 	if err := channel.DeleteState(closeSummary); err != nil {
-		req.Err <- err
-		return
+		if localReq != nil {
```

Comment:
The reason for this is that `handleClosingSigned` is called both by the initiator and the responder of the channel close request. In the case of responder, there won't be any `localReq`, and I set it to `nil`. Is there a better pattern we can use to share the common code between the two?